test/mpi/maint/test_coll_algos.sh

```diff
@@ -263,7 +263,9 @@ for algo_name in ${algo_names}; do
 
             echo "allgatherv2 10 ${env}" >> ${testlist_cvar}
             echo "allgatherv3 10 ${env}" >> ${testlist_cvar}
-            echo "allgatherv4 4 timeLimit=600 ${env}" >> ${testlist_cvar}
+            if ! [[ "$algo_name" =~ ^gentran_recexch_(doubling|halving) ]] ; then
+                echo "allgatherv4 4 timeLimit=600 ${env}" >> ${testlist_cvar}
```

Comment:
I just looked up this one and maybe we should leave it as it was, the test has 5 types of `recvcounts` array and I think it will work for some of them.